src/Configuration/Dotenv/Types.hs

```diff
@@ -0,0 +1,22 @@
+-- |
+-- Module      :  Configuration.Dotenv.Types
+-- Copyright   :  © 2015–2017 Stack Builders Inc.
+-- License     :  MIT
+--
+-- Maintainer  :  Stack Builders <hackage@stackbuilders.com>
+-- Stability   :  experimental
+-- Portability :  portable
+--
+-- Provides the types with extra options for loading a dotenv file.
+
+module Configuration.Dotenv.Types where
+
+-- | Alias for environment variables (NAME, VALUE).
+type Variable = (String, String)
```

Comment:
I'd prefer `(String, String)`. Reason: when I see `Variable` I think: "Hmm, and what *actually* this is?". I need to look up. With `(String, String)` I see what is this and how it's organized at a glance.